builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -17,6 +17,8 @@
 
 local store = { packages = {}, packages_full = {} }
 
+local http = minetest.request_http_api()
```

Comment:
does this return nil if not compiled with curl?
if yes that needs to be handled